File: test/samples/simple/simple.js
"use strict";

/**
 * @param {number} n the first number
 * @param {number} otherNumber the second number
 * @returns {void}
 */
function simple(n, otherNumber) {
    /* TODO Just do what you want with these numbers, like sum them. */
}

function main(stdin) {
    let line = 0;

    const n = Number(stdin[line++]);
    const otherNumber = Number(stdin[line++]);
    simple(n, otherNumber);
}

let stdin = "";
process.stdin.on("data", data => stdin += data.toString())
             .on("end", () => main(stdin.split("\n")));

File: test/samples/simple/simple.go
package main

import "bufio"
import "os"
import "strconv"

// n: the first number
// otherNumber: the second number
func simple(n int, otherNumber int) {
    /* TODO Just do what you want with these numbers, like sum them. */
}

func main() {
    scanner := bufio.NewScanner(os.Stdin)
    var n int
    scanner.Scan()
    n, _ = strconv.Atoi(scanner.Text())
    var otherNumber int
    scanner.Scan()
    otherNumber, _ = strconv.Atoi(scanner.Text())
    simple(n, otherNumber);
}
